docs/proto/data.md

```diff
@@ -0,0 +1,361 @@
+# Protocol Documentation
```

Comment:
instead of having one md per .proto file, i thought it was simpler to have one md per module

Reply:
Yep, definitely agree